Authentication/README.md

```diff
@@ -34,4 +34,12 @@ curl -v \
 ### Build and Deploy
 
      mvn clean install
-     mvn wildfly:deploy
\ No newline at end of file
+     mvn wildfly:deploy
+
+### Deploy on OpenShift
+
+To follow the instructions below, you need [rhc](https://developers.openshift.com/en/managing-client-tools.html) installed.
+
+     rhc app create bacon jboss-wildfly-8
+
+The ```bacon``` demo app is then available in [OpenShift online](http://bacon-corinnekrych.rhcloud.com/rest/grocery/bacons).
```

Comment:
@corinnekrych I think we should move the bacon app to the aerogear account. Lemme know what do you think.